server/fleet/users.go

```diff
@@ -27,6 +27,13 @@ type User struct {
 	Teams []UserTeam `json:"teams"`
 }
 
+func (u *User) IsAdminForcedPasswordReset() bool {
```

Comment:
Nit, but this could use a doc comment to explain why this is needed (I understand _what_ it does, but not sure _why_).